mirrord-common/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "mirrord-common"
```

Comment:
I think its better that the package will be only for the progress so changing it to mirrord-progress would be best